content/articles/getting-started-with-nlp-transformers/index.md

```diff
@@ -0,0 +1,193 @@
+﻿
+### Getting Started with NLP Transformers
+In today's world of daily new improvements and developments in technology, the field of Natural Language Processing (NLP) has not been left behind. New developments like Transfer Learning, Transformers (e.g., Google BERT & ELMO), Reinforcement Learning, Low-Code Tools etc. have changed the way data scientist work with textual data. They have also helped increase the area application for NLP like detecting fake news and monitoring social media for cases of cyber bullying.
+
+In this article, we will look at Transformers in NLP and their advantages over the traditional deep learning algorithms like recurrent neural networks (RNN). We will then learn how to build an NLP model using a transformer.
+
+### Introduction
+
```

Comment:
Ensure this spacing between the heading and first paragraph is removed throughout your article!

Reply:
> Ensure this spacing between the heading and first paragraph is removed throughout your article!

This comment here; FIX THE ENTIRE ARTICLE!